subprojects/test-runner/client/build.gradle.kts

```diff
@@ -11,36 +11,35 @@ publish {
 dependencies {
     compileOnly(gradleApi())
     api(projects.testRunner.service)
-    api(projects.common.result)
+    api(projects.testRunner.reportViewer) {
+        because("ReportViewerConfig exposes ReportCoordinates; also RunId")
+    }
 
-    implementation(projects.testRunner.testAnnotations)
-    implementation(projects.testRunner.instrumentationTestsDexLoader)
-    implementation(projects.testRunner.testReportArtifacts)
-    implementation(projects.testRunner.deviceProvider.impl)
-    implementation(projects.gradle.traceEvent)
-    implementation(projects.common.throwableUtils)
-    implementation(projects.common.math)
     implementation(projects.common.compositeException)
-    implementation(projects.common.result)
-    implementation(projects.common.problem)
+    implementation(projects.common.coroutinesExtension)
     implementation(projects.common.files)
-    implementation(projects.common.retrace)
-    implementation(projects.testRunner.fileStorage)
+    implementation(projects.common.math)
+    implementation(projects.common.problem)
+    implementation(projects.common.result)
+    implementation(projects.gradle.process)
+    implementation(projects.gradle.traceEvent)
+    implementation(projects.testRunner.deviceProvider.api)
+    implementation(projects.testRunner.deviceProvider.impl)
+    implementation(projects.testRunner.instrumentationTestsDexLoader)
+    implementation(projects.testRunner.reportProcessor)
```

Comment:
So could we make a next step and remove dependencies on `testReportArtifacts` and `reportProcessor`

Reply:
actually we can't
let's discuss it tomorrow